api/src/main/java/org/pmiops/workbench/dataset/DataSetMapper.java

```diff
@@ -37,17 +42,47 @@
   @Named("dbModelToClientLight")
   DataSet dbModelToClientLight(DbDataset dbDataset);
 
+  @Mapping(target = "dataSetId", ignore = true)
+  @Mapping(target = "version", source = "etag")
+  @Mapping(target = "creatorId", ignore = true)
+  @Mapping(target = "creationTime", ignore = true)
+  @Mapping(target = "invalid", ignore = true)
+  @Mapping(target = "lastModifiedTime", ignore = true)
+  @Mapping(target = "prePackagedConceptSetEnum", ignore = true)
+  @Mapping(target = "values", source = "domainValuePairs")
+  DbDataset dataSetRequestToDb(DataSetRequest dataSetRequest);
+
   @Mapping(target = "id", source = "dataSetId")
   @Mapping(target = "conceptSets", source = "conceptSetIds")
   @Mapping(target = "cohorts", source = "cohortIds")
   @Mapping(target = "domainValuePairs", source = "values")
   @Mapping(target = "etag", source = "version", qualifiedByName = "cdrVersionToEtag")
   DataSet dbModelToClient(DbDataset dbDataset);
 
+  default int etagToVersion(String eTag) {
+    return Strings.isNullOrEmpty(eTag) ? 1 : Etags.toVersion(eTag);
+  }
+
```

Comment:
Remove this and use qualifiedByName above. This method exists in CommonMappers.

Reply:
Done